pre-processing/primod/model_mixin.py

```diff
@@ -0,0 +1,27 @@
+"""Module containing mixins for specific kernels, for example MODFLOW."""
+
+from imod.mf6 import Modflow6Simulation, StructuredDiscretization
+from imod.mf6.mf6_wel_adapter import Mf6Wel
+
+
+class ModflowMixin:
+    """Modflow specific methods to add to coupling objects."""
+
+    @staticmethod
+    def get_mf6_pkgs_with_coupling_dict(
```

Comment:
Could you make this name more specific? We also couple to RIV, DRN and API packages in the Ribasim coupling for example. Those are not included here. Perhaps something like: get_mf6_pkgs_for_metaswap_coupling

Reply:
Good point! I didn't consider the Ribasim coupling when naming the method. Will fix.

Reply:
I followed your suggestion for the method name, and renamed ModflowMixin to MetaModMixin, as the method is not used for RibaMod.